### DISCUSSION TITLE
OSPOlogy Follow Up: Fostering FOSS Sustainability ThroughCollaborative Funding

### DISCUSSION THREAD

**Discussion by anajsana** [todogroup/ospology#305] [OSPO Stories]
Discuss with the community topics related to the upcoming OSPOlogy meeting:

📅 April, 26th ➡️ Fostering FOSS Sustainability ThroughCollaborative Funding

Join Zoom Link: TBD

In this open space, people can:

* Ask questions prior to the meeting starts
* Share notes from the conversation
* Make proposals for new initiatives that can help the OSPO Ecosystem
* Add useful links and resources related to the topic
* Connect with mentors involved in OSPOs and/or Open Source Management

[Slides](https://docs.google.com/presentation/d/1pZwrxvBIcv2Ibeoexike3hpr55GHDAcZK6OuU40mZfo/edit?usp=sharing) and [PDF](https://github.com/todogroup/ospology/files/11334114/OSPOlogy.S3.-.Chapter.3-1.pdf)

**Comment by Blackheart2** [comment]
Embajador ospo mexico

**Comment by anehzat** [comment]
We've been notified by several maintainers that they are being offered money to change their license https://os.cash/mission 
Is this a concern for the group?

**Reply by anehzat** [reply]
I did reach out to them to learn more about their intentions (license change) but no response yet.

**Reply by DuaneOBrien** [reply]
That's a new one for me. 

I don't see it as a huge concern, mainly because I don't think it will ultimately create any long term threat or change. If maintainers want to charge for their software, there are already mechanisms for doing so that don't involve the sticky problems associated with relicensing their projects. This reads more like a lawyer trying to create their own business opportunity than a solution to an actual problem.

**Comment by souravcipher** [comment]
1. What are some of the biggest challenges that OSPOs face when it comes to funding FOSS projects sustainably?
2. How can OSPOs encourage more diversity and inclusion in FOSS projects, and what role does funding play in this effort?

**Reply by DuaneOBrien** [reply]
One of the biggest problems I see is identifying projects that will benefit the most from funding, versus projects that have good funding governance, versus projects that have strong name recognition and effective fundraising operations. I still think we're learning how to find the most impactful funding opportunities.

As for improving the flow of funding with an eye toward diversity and inclusion, I would say that like any other effort that wants to make an impact in these areas, it takes intentional work. You have to dig into the projects and the people behind them to find those opportunities to make improvements in how you're distributing funds. There's a really important discussion to be had here about "funding projects" versus "funding people" that we didn't get into today.

By way of example - when I was at Indeed, the first time I did a deep analysis of the contributors to our key open source projects, the framework I'd developed produced a set of sponsorship recommendations that only showed me people presenting as male, almost all of them in the US, and no people of color. In order to diversify the flow of funding, I needed to do additional analysis to find and elevate women and contributors of color. In some cases, we were their first sponsors. That can send an important signal that helps them land their 2nd and 3rd sponsors, which can help. I wouldn't ever say that my approach was perfect though, and I think we all still have a lot of learning to do in this area.

**Reply by souravcipher** [reply]
Thanks for sharing your insights @DuaneOBrien, really appreciate it!

**Comment by anajsana** [comment]
new question from the chat

"Since open source software is getting recognized as a digital infrastructure, what do you think about establishing open source funds at a national level, similar to Sovereign Tech Fund in Germany?"

**Reply by anajsana** [reply]
from the chat: "How the United Nations is approaching digital public goods is also interesting https://www.un.org/techenvoy/content/digital-public-goods"

**Reply by coni2k** [reply]
Thanks for adding the question here, @anajsana. Let me also add the Sovereign Tech Fund link:
https://sovereigntechfund.de/en/

**Reply by chadwhitacre** [reply]
These are great, thanks for bringing them up! Open source is both a public good (code itself) and a common-pool resource (maintainer attention). The historically successful mechanism for funding either is taxation by government or quasi-government, so that's likely also what's needed here. At the end of the day, depending on companies to voluntarily fund open source at scale is unlikely to be successful.

**Reply by coni2k** [reply]
I agree that we should recognize open source software as a public good and finance it with public money; addressing the funding problem at a national level should be the most optimum solution in the long term instead of expecting each company to take action voluntarily. And it's pretty exciting to hear these from you.

I'd love to see a solid effort within the open source community to encourage governments to establish such national public funds.

What do you think we should do to move this conversation forward? Would it be practical to form a working group?

**Reply by coni2k** [reply]
Let me also attach the humble funding experiment that I've been running for 15 months to study, demo, and increase awareness around public funds:
https://dev.to/coni2k/open-source-public-fund-experiment-lc8

**Comment by anajsana** [comment]
new question from the chat:

How do we align with the interests of shareholders for billion dollar companies who see open source support as altruism perhaps, or open-washing? Open source licensing says it’s free, but is it really?

**Reply by anajsana** [reply]
from the chat:

It's a continual conversation of describing business value as Alex said ( using language that matters at that level: security, innovation, diverse perspectives, quality).  The truth is not everyone comes into a company understanding or having any experience learning.  It's absolutely a role an OSPO can play.

**Comment by anajsana** [comment]
new question from the chat:

Once you decide which OSS project(s) to sponsor, what is typically the process of providing the funding to the project, what mechanisms do you use for that, and what obstacles do you face?

**Reply by chadwhitacre** [reply]
At Sentry we support a handful of foundations directly, but for the long tail we only support projects that are on one of a few platforms (GitHub Sponsors and Open Collective primarily; this past year we piloted Thanks.dev and StackAid). We have to go through our normal procurement process to add these platforms as vendors.

**Comment by itprofjacobs** [comment]
Last day of HFOSS Class is tomorrow.  Thanks for providing an entertaining discussion topic to close out the semester :-)
